diff --git a/data/com.stremio.Stremio.service b/data/com.stremio.Stremio.service
index 7cfb139..c3282fa 100644
--- a/data/com.stremio.Stremio.service
+++ b/data/com.stremio.Stremio.service
@@ -1,3 +1,3 @@
 [D-BUS Service]
 Name=com.stremio.Stremio
-Exec=/app/bin/stremio --gapplication-service
+Exec=/usr/bin/stremio --gapplication-service
diff --git a/src/config.rs b/src/config.rs
index 6f40e84..df97f4a 100644
--- a/src/config.rs
+++ b/src/config.rs
@@ -2,7 +2,7 @@ pub const DATA_DIR: &str = "stremio";
 
 pub const GETTEXT_DOMAIN: &str = "stremio";
-pub const GETTEXT_DIR_DEV: &str = concat!(env!("CARGO_MANIFEST_DIR"), "/po");
+pub const GETTEXT_DIR_DEV: &str = "/usr/share/locale";
 pub const GETTEXT_DIR_FLATPAK: &str = "/app/share/locale";
 
 pub const STARTUP_URL: &str = "http://127.0.0.1:11470/proxy/d=https%3A%2F%2Fweb.stremio.com/";
